openapi-diff/src/modeler/AutoRest.Swagger/Model/Response.cs

```diff
@@ -66,19 +66,20 @@ public override IEnumerable<ComparisonMessage> Compare(ComparisonContext context
             {
                 context.PushProperty(header.Key);
 
-                Header newHeader = null;
-                if (!headers.TryGetValue(header.Key, out newHeader))
+                if (!headers.TryGetValue(header.Key, out var newHeader))
                 {
                     context.LogBreakingChange(ComparisonMessages.RemovingHeader, header.Key);
                 }
 
                 context.Pop();
             }
 
+            context.PushProperty("schema");
```

Comment:
could you describe a bit the logic being used pushing properties to compare?

Reply:
it was a bug. the code creates an ObjectPath by pushing properties to the context. We have to push `schema` here.